platform/nservicebus.include.md

```diff
@@ -1,5 +1,68 @@
-NServiceBus, the heart of the system, is a messaging and workflow framework. It helps create distributed systems that are scalable, reliable, and easy to change. It supports various messaging patterns, encapsulates long-running business processes as [sagas](/nservicebus/sagas), and provides abstractions over several [queuing](/transports/) and [storage](/persistence/) technologies.
+NServiceBus is the heart of a distributed system and the Particular Service Platform. It helps create systems that are scalable, reliable, and flexible.
 
-Most queuing technologies try to guarantee 'at least once', or even 'exactly once', message delivery. They usually fall short of this promise. NServiceBus solves intermittent delivery problems by  automatically retrying messages using various strategies. If all else fails, it forwards messages to an error queue. From there, [ServiceControl](/servicecontrol/) and [ServicePulse](/serviceinsight/) expose the problems for human intervention.
+At its core, NServiceBus works by routing _messages_ between _endpoints_. [Messages](/nservicebus/concept-overview.md#message) are plain C# classes that contain meaningful data for the business process that is being modeled.
 
-NServiceBus is modular in design and highly extensible. There are many options to choose from for queueing, storage, serialization, logging, and more. Many elements of the system can be replaced with custom implementations for specific scenarios.
+```csharp
+public class ProcessOrder : IMessage
```

Comment:
I would consider showing unobtrusive here, which should probably be more strongly advertised as the default throughout our docs. I know for a fact that some developers find inheriting from a base type for a message highly off-putting, and it may not be obvious that it's optional.

```suggestion
public class ProcessOrder
```